db/migrate/20150128155644_allow_null_for_adviser_covers_whole_of_uk.rb

```diff
@@ -0,0 +1,5 @@
+class AllowNullForAdviserCoversWholeOfUk < ActiveRecord::Migration
+  def change
+    change_column :advisers, :covers_whole_of_uk, :boolean, null: true, default: nil
```

Comment:
Relying on the DB here isn't necessary as others pointed out. We should replace this with non-null and default the column. This kind of thing is perfect for `after_initialize` as @jongilbraith pointed out.